components/Buttons/src/MDCButton.m

```diff
@@ -546,7 +546,8 @@ - (UIColor *)backgroundColorForState:(UIControlState)state {
   if ((state & UIControlStateHighlighted) == UIControlStateHighlighted) {
     state = state & ~UIControlStateDisabled;
   }
-  return _backgroundColors[@(state)];
+
+  return _backgroundColors[@(state)] ?: _backgroundColors[@(UIControlStateNormal)];
```

Comment:
I wonder if this is correct for the highlighted color. Are we expecting the highlighted background color to be same as the ink color?

Reply:
I'm not sure I follow - where did you get the Ink color from?